.cirrus/tasks_sonarqube_dce.yml

```diff
@@ -1,5 +1,5 @@
 SKIP_IF_NOT_SONARQUBE_DCE: &SKIP_IF_NOT_SONARQUBE_DCE
-  skip: "!changesInclude('charts/sonarqube-dce/*', 'charts/sonarqube-dce/**/*', 'tests/unity-compatibility-test/sonarqube-dce/*', '.cirrus/*.sh', '.cirrus/tasks{_sonarqube_dce,_templates,}.yml', '.cirrus/tasks.star', '.cirrus.star') && $CIRRUS_CRON != $NIGHTLY_CRON"
+  skip: "!($CIRRUS_PRERELEASE != \"true\" && $CIRRUS_RELEASE != \"\") && !changesInclude('charts/sonarqube-dce/*', 'charts/sonarqube-dce/**/*', 'tests/unity-compatibility-test/sonarqube-dce/*', '.cirrus/*.sh', '.cirrus/tasks{_sonarqube_dce,_templates,}.yml', '.cirrus/tasks.star', '.cirrus.star') && $CIRRUS_CRON != $NIGHTLY_CRON"
```

Comment:
always hard to read those skip with negation (!) could we revert it for more clarity ? 

That would be (to be verified)

skip: "$CIRRUS_RELEASE == \"\"" and we dont need the CIRRUS_PRERELEASE as they are mutualy exclusive ? 

wdyt ?